ooniprobe/Engine/Engine.m

```diff
@@ -1,6 +1,6 @@
 #import "Engine.h"
 #import <mkall/MKVersion.h>
-#define probeEngineTasks @[@"Telegram", @"Ndt", @"Dash", @"Psiphon", @"Tor"]
+#define probeEngineTasks @[@"Telegram", @"Ndt", @"Dash", @"Psiphon", @"Tor", @"Whatsapp", @"FacebookMessenger", @"HttpHeaderFieldManipulation", @"HttpInvalidRequestLine"]
```

Comment:
I know it's a long line! splitting it caused me problems as it's a define

Reply:
never mind